controllers/apps/systemaccount_controller_test.go

```diff
@@ -99,6 +99,8 @@ var _ = Describe("SystemAccount Controller", func() {
 		inNS := client.InNamespace(testCtx.DefaultNamespace)
```

Comment:
missing bug fix codes?

Reply:
fix is around line 648 and 661. updated the job to be deleted, and remove finalizer for jobs.